### ISSUE TITLE
Dual Wield Critical Strike Multiplier and Accuracy [broken]

### ISSUE THREAD

**Issue by 0ldi** [PathOfBuildingCommunity/PathOfBuilding#890]
![image](https://user-images.githubusercontent.com/16578229/81505822-df0ef480-92fa-11ea-92c7-bbf24ecd56ab.png)

Red, as I understand it, is not calculating at all.
Probably it's GGG text bug because exist (working at PoB):
[to Critical Strike Multiplier while Dual Wielding](https://pathofexile.gamepedia.com/Modifier:DualWieldingCritMultiplierJewel)
not working instead:
[to **Attack** Critical Strike Multiplier while Dual Wielding](https://pathofexile.gamepedia.com/Dual_Wield_Critical_Strike_Multiplier_and_Accuracy)

**Comment by AdmiralChopper** [comment]
Managed to fix it by adding this line on ModParser-3_0
```
["attack critical strike multiplier"] = { "CritMultiplier", flags = ModFlag.Attack },
```
And rebuilding the ModCache by holding ctrl while launching. But since the ModCache is changed only on new releases, I think this will get fixed only on next release?

**Comment by LocalIdentity** [comment]
Fixed in 1.4.170.1
